src/plugins/semantic-release/index.ts

```diff
@@ -0,0 +1,51 @@
+import path from 'node:path';
+import { _load } from '../../util/loader.js';
+import { timerify } from '../../util/performance.js';
+import { hasDependency } from '../../util/plugin.js';
+import type { PluginConfig } from './types.js';
+import type { IsPluginEnabledCallback, GenericPluginCallback } from '../../types/plugins.js';
+
+// link to docs
+
+export const NAME = 'Semantic Release';
+
+/** @public */
+export const ENABLERS = ['semantic-release'];
+
+export const isEnabled: IsPluginEnabledCallback = ({ dependencies }) => hasDependency(dependencies, ENABLERS);
+
+// https://github.com/semantic-release/semantic-release/blob/master/docs/usage/configuration.md#configuration-file
+export const CONFIG_FILE_PATTERNS = [
+  '.releaserc',
+  '.releaserc.{yaml,yml,json,js,cjs}',
+  'release.config.{js,cjs}',
+  'package.json',
+];
+
+export const ENTRY_FILE_PATTERNS = [];
+
+export const PRODUCTION_ENTRY_FILE_PATTERNS = [];
+
+export const PROJECT_FILE_PATTERNS = [];
+
+async function load(filePath: string) {
```

Comment:
Other plugin might also need this load behavior (where the file can be either json or yaml when there is no extension) so this might want to be refactor out. Maybe `_load` itself should just handle this.